### PR TITLE
(TK-383) Initial project skeleton

### PR DESCRIPTION
**Pull request by haus**
This PR adds a CONTRIBUTING.md to describe the contribution process as well as adds a maintainers section to the README.md.

### CODE REVIEW

**Comment by haus** [comment]
The travis failure is expected until there is a project.clj

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: clojure
+lein: lein2
+jdk:
+  - oraclejdk7
+  - openjdk7
```
Comment (KevinCorcoran) [inline .travis.yml:5]: I realize this is probably copied from another project, but ...
- We should add java 8 if possible
- I don't see why we would need to test against two versions of java 7, especially considering we have Jenkins.

If those things both make sense, we can probably just change `oraclejdk7` to `oraclejdk8` and call it good.

File: project.clj [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  :exclusions [org.clojure/clojure]
+
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/tools.logging "0.3.1"]
+                 [prismatic/schema "1.1.1"]
+                 [me.raynes/fs "1.4.6"]
+                 [puppetlabs/trapperkeeper ~tk-version]
+                 [puppetlabs/kitchensink ~ks-version]
+                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
```
Comment (KevinCorcoran) [inline project.clj:20]: hey, ya know what's fun?
1. we've only ever done a single release of tk-sched.  yey?
2. there's no tag for it on github.  booo.    https://github.com/puppetlabs/trapperkeeper-scheduler/releases

**Comment by KevinCorcoran** [comment]
👍 , going to merge and work on top of this, but we should still update travis to `oraclejdk8` if possible

**Comment by KevinCorcoran** [comment]
In retrospect, I might have let https://github.com/puppetlabs/ci-job-configs/pull/1492 land before merging this.  Oh well, YOLO.

**Comment by haus** [comment]
Both of your comments make sense. If you haven't gotten to it I'll put up a quick PR to fix it up to day.

**Comment by KevinCorcoran** [comment]
Cool, I've got the bump to jdk 8 in a branch I'm working on for the other thing.

**Pull request merged**